src/resources/views/crud/fields/select2.blade.php

```diff
@@ -67,13 +67,16 @@
         <!-- include select2 js-->
         <script src="{{ asset('packages/select2/dist/js/select2.full.min.js') }}"></script>
         @if (app()->getLocale() !== 'en')
-        <script src="{{ asset('packages/select2/dist/js/i18n/' . app()->getLocale() . '.js') }}"></script>
+        <script src="{{ asset('packages/select2/dist/js/i18n/' . str_replace('_', '-', app()->getLocale()) . '.js') }}"></script>
         @endif
         <script>
             function bpFieldInitSelect2Element(element) {
                 // element will be a jQuery wrapped DOM node
                 if (!element.hasClass("select2-hidden-accessible")) {
                     element.select2({
+@if (app()->getLocale() !== 'en')
+                        language: "{{ $str_replace('_', '-', app()->getLocale()); }}",
+@endif
```

Comment:
@shiroamada is this part necessary?
Because until now we didn't use it, by only loading the script of the language `.../i18n/pt.js` it would load the language.
Can you confirm this?

Reply:
Yes, it is necessary for zh_CN, i tried, if I didn't add the line, it will not load it.

This is did not add the `language:` parameter.
![image](https://user-images.githubusercontent.com/2062946/110330497-c6758500-8058-11eb-85e7-613e286a379e.png)

This is added the `language:` parameter.
![image](https://user-images.githubusercontent.com/2062946/110330372-a47c0280-8058-11eb-99c5-ea0aef13ff10.png)

Maybe you can check your pt.js is it working when drop down?

Reply:
Hi @shiroamada, you were right, without that it doesn't work. Meanwhile I moved it to element attributes.
Thank you for you help 🙌